docs/victorialogs/security.md

```diff
@@ -0,0 +1,523 @@
+If using self-signed certificates, you have two options:
+1. Install your CA certificate in the system's trusted certificate store.
+1. Specify the CA certificate path manually using `-remoteWrite.tlsCAFile`.
+
+For testing purposes only, you can disable certificate verification with `-remoteWrite.tlsInsecureSkipVerify`.
+
+**Security Warning:** Disabling certificate verification eliminates
+protection against man-in-the-middle attacks. Never use this in production.
+
+### Automatic issuing of TLS certificates
+
+All the [VictoriaLogs Enterprise](https://docs.victoriametrics.com/victoriametrics/enterprise/) components support automatic issuing of TLS certificates
+for public HTTPS server running at `-httpListenAddr` via [Let's Encrypt service](https://letsencrypt.org/).
+The following command-line flags must be set in order to enable automatic issuing of TLS certificates:
+
+- `-httpListenAddr` must be set for listening TCP port `443`. For example, `-httpListenAddr=:443`.
+  This port must be accessible by the [Let's Encrypt service](https://letsencrypt.org/).
+- `-tls` must be set in order to accept HTTPS requests at `-httpListenAddr`.
+  Note that `-tlcCertFile` and `-tlsKeyFile` aren't needed when automatic TLS certificate issuing is enabled.
+- `-tlsAutocertHosts` must be set to comma-separated list of hosts, which can be reached via `-httpListenAddr`.
+  TLS certificates are automatically issued for these hosts.
+- `-tlsAutocertEmail` must be set to contact email for the issued TLS certificates.
+- `-tlsAutocertCacheDir` may be set to the directory path for persisting the issued TLS certificates between VictoriaMetrics restarts.
+  If this flag isn't set, then TLS certificates are re-issued on every restart.
+
+Example of starting VictoriaLogs with automatic TLS certificate issuing:
+
+```sh
+./victoria-logs -httpListenAddr=:443 \
+    -tls \
+    -tlsAutocertHosts=victorialogs.example.com,logs.example.com \
+    -tlsAutocertEmail=admin@example.com \
+    -tlsAutocertCacheDir=/path/to/tls-cache \
+    -licenseFile /path/to/license
+```
+
+#### The `no viable challenge type found` error
+
+Let's Encrypt validates domain ownership by performing HTTP-01 challenges to verify you control the specified domains.
+The Let's Encrypt service must be able to reach your VictoriaLogs instance over the public internet on port 443 to complete this validation.
+
+Common troubleshooting steps:
+1. Ensure your server is reachable from the internet on port 443
+1. Confirm that your domain names resolve to your server's public IP address
+1. Use external tools to verify HTTP connectivity
+
+Example connectivity verification:
+```sh
+# Test domain resolution
+nslookup victorialogs.example.com
+
+# Test HTTP connectivity from external network
+curl -I http://victorialogs.example.com/.well-known/acme-challenge/test
+```
+
+This functionality can be evaluated for free according to [these docs](https://docs.victoriametrics.com/victoriametrics/enterprise/).
```

Comment:
This phrase was copied from another page and I can't find any information about free automatically issued TLS certificates. I suggest to remove this. Or I miss something?